cluster-autoscaler/cloudprovider/azure/azure_agent_pool_test.go

```diff
@@ -105,3 +109,23 @@ func TestDeleteOutdatedDeployments(t *testing.T) {
 		assert.Equal(t, test.expectedDeploymentsNames, existedDeploymentsNames, test.desc)
 	}
 }
+
+func TestGetVirtualMachinesFromCache(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	defer ctrl.Finish()
+
+	testAS := newTestAgentPool(newTestAzureManager(t), "testAS")
```

Comment:
should it be `testAP` or is `AS` for availbilityset?

Reply:
yeah as is for availbilityset.